config/routes.rb

```diff
@@ -7,7 +7,7 @@
 
    resources :restaurants
 
-  
+   resources :menus
```

Comment:
You should remove this line since you're not using the User model & controller for this branch